src/cloudformation_cli_python_lib/metrics.py

```diff
@@ -0,0 +1,96 @@
+import logging
+from typing import Any, Mapping
+
+# boto3 doesn't have stub files
+from boto3.session import Session  # type: ignore
+
+from botocore.exceptions import ClientError  # type: ignore
+
+from .interface import Action, MetricTypes, StandardUnit
+
+LOG = logging.getLogger(__name__)
+
+
+def format_dimensions(dimensions: Mapping[Any, Any]) -> Any:  # TODO fix type
```

Comment:
from https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/cloudwatch.html#CloudWatch.Client.put_metric_data it looks like key and value must both be strings, so `Mapping[Str, Str] -> List[Mapping[Str, Str]]` should do the trick here

Reply:
updated